src/keystore.h

```diff
@@ -23,7 +23,6 @@ class CKeyStore
 {
 protected:
     mutable CCriticalSection cs_KeyStore;
-    mutable CCriticalSection cs_SpendingKeyStore;
```

Comment:
With this being removed, the PR would fail to compile if any `cs_SpendingKeyStore` had been missed.